WordPress/Classes/Services/MediaImageService.swift

```diff
@@ -244,6 +351,9 @@ extension MediaImageService {
         case .medium:
             let side = min(1024, minScreenSide * UIScreen.main.scale)
             return CGSize(width: side, height: side)
+        case .original:
+            assertionFailure("Unsupported thumbnail size")
+            return CGSize(width: 2048, height: 2048)
```

Comment:
I throw an error earlier:, so technically this code is unreachable.

```swift
        guard size != .original else {
            assertionFailure("Unsupported thumbnail size: \(size)")
            throw Error.unsupportedThumbnailSize(size)
        }
```

This function isn't throwing, so I'd prefer to keep it as is. It's a programmatic error to call it, so maybe it should even be promoted to a fatalError. I think it's fine either way since it's not reachable.

Reply:
That guard is in a different function, right? It's still possible to call `getPreferredThumbnailSize(for: .original)`, which wouldn't be an issue at all. IMO it's okay to use `fatalError` if you think the statement is not reachable at runtime.